### DISCUSSION TITLE
v2.0.0rc2: Update changelog

### DISCUSSION THREAD

**Discussion by miurahr** [miurahr/aqtinstall#376] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/miurahr/aqtinstall/releases/tag/v2.0.0rc2'>v2.0.0rc2: Update changelog</a>.</em>

**Comment by miurahr** [comment]
Pending tasks

- [ ] Update README
  ref  [comment agianst v2.0.0rc1 discussion](https://github.com/miurahr/aqtinstall/discussions/367#discussioncomment-1192567)

**Reply by pzhlkj6612** [reply]
Oh, hi! I have a new question about it: do we really need to keep a series of documentation sections in README?

I don't think so. My points:

1. Users can get started from https://aqtinstall.readthedocs.io/ .
2. We can avoid repetitive work related to document update.

Looking forward to your opinion.

**Reply by ddalcino** [reply]
I think we do need instructions on how to use aqt in the README. We could try to cut it down to the bare minimum though, and move discussion of the more advanced features to the 'readthedocs' site.

There are several things that a README needs to have, and one of those things is instruction on basic usage. You can google "What to put in my readme" and find tons of different opinions on the subject, but "Instruction on basic usage" is common to all of the opinions I looked at. Basic instructions can help a new user figure out whether or not the project will work for their use case, and it helps to know that information without having to dig it out of the manual.

Here's an opinion on READMEs from https://liw.fi/readme-review/ that I find particularly helpful:

>* A README is meant for someone who is first encountering the project to be able to quickly decide if they want to learn more about it.
>* A README should have a “blurb”: a very short, dense summary of the project aimed at someone who doesn’t know anything about it yet and may not know the topic area.
>* A README is not the manual, but it may have an example of how to typically use the software, especially if that helps the reader understand the software better.
>* A README should say how the copyright and other legal aspects affect the project. It should say what copyright licenses the project uses.
>* A README can also contain instructions for installing the software, or how to run its test suite, and other practical aspects, but those can go into other documentation as well. Referring to them is good, as is linking to more information about the project, and who works on the software.

I think that anything that's not on this list is a good candidate to move to the 'readthedocs' site.

**Reply by miurahr** [reply]
There are candidates to remove from README.

- Usage(as in v1.2.x)
- Installing tool and utility (Experimental)
- Target directory
- Supported CI plarform
- (Advanced) Force dependency
- (most of) command examples

We can add a link to  `getting started` section of manual
and  add a simple `list-qt` and `install-qt` example to illustrate typical usage.

**Reply by miurahr** [reply]
Here is a proposal: https://github.com/miurahr/aqtinstall/pull/377

**Reply by miurahr** [reply]
You can check it at a glance 
 https://github.com/miurahr/aqtinstall/tree/topic-readme-shorter-dense

**Comment by miurahr** [comment]
README.rst is also used as PyPI page.

Here is a development head: https://test.pypi.org/project/aqtinstall/2.0.0rc2.dev48/

It seems PyPI document tool don't render ``.. warning`` and ``.. note``

**Reply by pzhlkj6612** [reply]
> It seems PyPI document tool don't render `.. warning` and `.. note`

See https://github.com/pypa/readme_renderer/issues/132 .
